source/Mlos.Notebooks/BayesianOptimization.ipynb

```diff
@@ -485,7 +485,7 @@
     "# evaluate the surrogate\n",
     "surrogate_predictions = optimizer.predict(pd.DataFrame({'x': line})).get_dataframe()\n",
     "# plot observations\n",
-    "feature_values, target_values = optimizer.get_experiment_data()\n",
+    "feature_values, target_values = optimizer.get_all_observations()\n",
```

Comment:
This is the main substantial change in this PR, the method was renamed.

Reply:
This type of thing should get caught when we finish #55